beacon-chain/blockchain/process_block.go

```diff
@@ -405,6 +403,25 @@ func (s *Service) onBlock(ctx context.Context, signed interfaces.SignedBeaconBlo
 	return nil
 }
 
+func (s *Service) tryPublishLightClientFinalityUpdate(ctx context.Context, signed interfaces.SignedBeaconBlock, finalized *forkchoicetypes.Checkpoint, postState state.BeaconState) {
+	if finalized.Epoch > s.lastPublishedLightClientEpoch {
+		config := params.BeaconConfig()
+		if finalized.Epoch > config.AltairForkEpoch {
+			syncAggregate, err := signed.Block().Body().SyncAggregate()
+			if err == nil {
+				if syncAggregate.SyncCommitteeBits.Count()*3 >= config.SyncCommitteeSize*2 {
+					_, err := s.sendLightClientFinalityUpdate(ctx, signed, postState)
```

Comment:
should we roll this logic inside sendLightClientFinalityUpdate() instead of having the try wrapper?